components/modal/src/modal/modal.js

```diff
@@ -7,67 +7,60 @@ import PropTypes from 'prop-types'
 import React from 'react'
 import { resolve } from 'styled-jsx/css'
 
-const scrollBoxCard = resolve`
-    div {
-        display: flex;
-        flex-direction: column;
-        max-height: calc(100vh - ${2 * spacersNum.dp64}px);
-        max-width: calc(100vw - ${2 * spacersNum.dp64}px);
-    }
-`
-
-const centeredContent = resolve`
-    .top {
-        padding-top: ${spacers.dp64};
-    }
-    .bottom {
-        padding-bottom: ${spacers.dp64};
-    }
-`
-
-const layerStyles = resolve`
-    div {
-		display: none;
-    }
-`
+const resolveLayerStyles = (hide) =>
+    resolve`
+        div {
+            padding: ${spacers.dp64};
```

Comment:
@cooper-joe I wonder if we should make this the default here: https://github.com/dhis2/ui/blob/master/components/layer/src/layer.js#L40

That would stop anything that is rendered on top of the Layer to cover the entire viewport.

Reply:
Yes, that sounds reasonable. That padding doesn't really concern the `modal`, it's more of a display property of the `Layer`s contents. So as a principle I support this change. Only issue I can see is some unknown use cases of the `Layer` that will be broken or mangled by the change.

Reply:
Yeah, we'll have to investigate in a separate PR. Leaving this as is in this, but wanted to pointed it out.